frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -413,6 +415,24 @@ private void serveAddVendor(ResourceRequest request, ResourceResponse response)
             log.error("Error adding vendor", e);
         }
     }
+    
+    private void serveViewDepartment(ResourceRequest request, ResourceResponse response) throws IOException, PortletException {
+        renderDepartmentSearch(request, response);
+    }
+    
+    private void renderDepartmentSearch(ResourceRequest request, ResourceResponse response) throws IOException, PortletException {
```

Comment:
This method is identical in `ProjectPortlet.java` as well. Please check if it can  be placed in any common place.

Reply:
I set it in common place, PortletUtils

Reply:
@smrutis1  could you check it again?